environment.yml

```diff
@@ -1,21 +1,20 @@
 name: cyto_39
 channels:
+  - pytorch
   - conda-forge
+  - defaults
 dependencies:
-  - numpy
+  - python=3.9
+  - pytorch=1.10.0
+  - mkl=2024.0
+  - chromadb=0.5.3
```

Comment:
I liked the suggestion in #14 of loading all this into `pyproject.toml` instead but this is a nice increment, thank you for the cleanup job!